src/App.tsx

```diff
@@ -34,9 +38,17 @@ export function getReorderedGoods(
   // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  visibleGoods.sort((goodA, goodB) => {
+    switch (sortType) {
+      case SortType.ALPABET: return goodA.localeCompare(goodB);
+      case SortType.LENGTH: return goodA.length - goodB.length;
```

Comment:
```suggestion
      case SortType.ALPABET: 
        return goodA.localeCompare(goodB);
        
      case SortType.LENGTH: 
        return goodA.length - goodB.length;
```

it's hard to read, better to code this way